src/main/java/team/creative/littletiles/client/tool/LittleToolPlacer.java

```diff
@@ -410,7 +412,10 @@ public boolean onRightClick(Level level, Player player, BlockHitResult result) {
         if (!built)
             return false;
         if (!checkForWorker())
+        {
+            if (worker == null) return false;
             builtResult = worker.join();
```

Comment:
Oh alright, if there is nothing to place, worker will probably be null and in that case nothing should happen. Could also be replaced with builtEmpty, but I guess it does not matter.